src/code42cli/profile.py

```diff
@@ -102,7 +102,8 @@ def switch_default_profile(profile_name):
 def create_profile(name, server, username, ignore_ssl_errors):
     if profile_exists(name):
         raise Code42CLIError("A profile named '{}' already exists.".format(name))
-
+    if None in {name, server, username}:
```

Comment:
This causes the `[required]` documentation help text to go away from the `create` command, where these are actually required. An alternative approach that will allow the help text to remain accurate would be to use parameterized methods for creating the `click.options`.

Example:

```
def usename_option(required=False):
  return click.option(
    "-u", "--username", required=required, help="The username of the Code42 API user.",
```

Then, in the update function, use it like this:

```
@profile.command()
...
@username_option()
def update()...
```

And in the create method,

```
@profile.command()
...
@username_option(required=True)
def create()...
```